extension/lib/webcompat.js

```diff
@@ -85,20 +86,46 @@ class WebCompat {
   /**
    * @param {String} elementName -
    *                e.g. div, main, aside
+   * @param {Array} attributes -
+   *                e.g. [{ name: "dir", value: "ltr" }, ...]
    * @param {Array} browsers -
    *                e.g. [{ id: "firefox", name: "Firefox", version: "68" }, ...]
-   * @return {Object} if no issue found, return null.
+   * @return {Array} if there are no issues found, then the array is empty.
    */
-  getHTMLElementIssue(elementName, browsers) {
-    const database = this._webCompatData.html.elements;
-    const summary = this._getCompatSummary(browsers, database, elementName);
+  getHTMLElementIssues(elementName, attributes, browsers) {
+    const databaseElements = this._webCompatData.html.elements;
+    const databaseGlobalAttributes = this._webCompatData.html.global_attributes;
+    const issues = [];
 
-    if (!this._hasIssue(summary)) {
-      return null;
+    const elementSummary =
+      this._getCompatSummary(browsers, databaseElements, elementName);
+    if (this._hasIssue(elementSummary)) {
+      elementSummary.element = elementName;
+      issues.push(this._toIssue(elementSummary, _ISSUE_TYPE.HTML_ELEMENT));
     }
 
-    summary.element = elementName;
-    return this._toIssue(summary, _ISSUE_TYPE.HTML_ELEMENT);
+    for (const { name: attributeName, value: attributeValue } of attributes) {
+      let attributeSummary = null;
+      if (this._hasTerm(databaseElements, elementName, attributeName)) {
+        attributeSummary =
+          this._getCompatSummary(browsers, databaseElements, elementName, attributeName);
+      } else {
+        const keyword =
+          attributeName.startsWith("data-") ? "data_attributes" : attributeName;
+        attributeSummary =
+          this._getCompatSummary(browsers, databaseGlobalAttributes, keyword);
+      }
+
+      // Don't apply the invalid attribute which was not in the database as issue.
```

Comment:
Could you add a second comment line like:

`// Because frameworks like angular make heavy use of custom attributes`